telemetry/csharp/AwsToolkit.Telemetry.Events.Generator/DefinitionsBuilder.cs

```diff
@@ -213,7 +213,7 @@ private CodeTypeDeclaration GenerateEnumStruct(MetricType type)
                     allowedValue.Replace(" ", "_").ToPascalCase().Replace(".", "").Replace("-", ""))
                 {
                     InitExpression = new CodeObjectCreateExpression(type.GetGeneratedTypeName(),
```

Comment:
For C# we converted the "key" but not the "value" to get rid of spaces. This was an oversight and testing of course caught this. 

Thanks for pushing for testing.

Reply:
Were any Toolkits released with spaces in the values? This change would create a discrepancy in emitted values across Toolkit versions.

Why not change the definition data to remove the spaces? I don't understand why the values cannot contain spaces.

Reply:
There were spaces deployed this week. But we can revert this quickly as our slack discussion aligns with this being better for backend ingestion now and in the future.

Reply:
should we validate the `allowedValues` so that CI fails if they contain spaces?